### ISSUE TITLE
External Storage Shares Nextcloud. 

### ISSUE THREAD

**Issue by mickeyace** [nextcloud/server#18078]
Recently my nextcloud has stopped allowing adding any new External Shares and it no longer allows the ability to add new users to these external shares. I am assuming this is related to an update but I have been able to locate a suitable fix for the issue. Any assistance would be greatly appreciated.  
Nextcloud External shares that were already setup continue to work without issue. Users who were part of the shares are still working but I am unable to add new shares or add new users to those external shares. 

## Steps to reproduce
1.
2.
3.

## Expected behaviour
Tell us what should happen

## Actual behaviour
Tell us what happens instead

## Server configuration

**Nextcloud server version:** (see your admin page)

**Server OS** (Ubuntu server is default)

**How did you install the VM?** (Scripted install from master **OR** Released version)

## Network
**Do you use DHCP?**

**Is port 80 and/or 443 open?**

#### Log file (/mnt/ncdata/nextcloud.log)
```
Insert your nextcloud.log file here
```

#### Installation log (command output)
```
Insert the CLI output here
```

**Comment by enoch85** [comment]
Thanks for your issue @mickeyace.

Please fill in the server issue template for this issue.

**Comment by StodgyWaif** [comment]
I'm having the same issue on one of my instances.
Ubuntu 18.04.3
PHP 7.2.24
Apache 2.4.29
NextCloud 17.0.1
External Storage support 1.8.0
I've even attempted to just copy one of the currently working external mount's settings to a new one (with a different label) and it fails with a red ! symbol. Does not matter which type of credentials I use.

**Comment by kesselb** [comment]
Duplicate of https://github.com/nextcloud/server/issues/17846

Will be fixed with 17.0.2 and 16.0.7
